exporter/kafkaexporter/config.go

```diff
@@ -103,7 +105,7 @@ func (cfg *Config) Validate() error {
 	return validateSASLConfig(cfg.Authentication.SASL)
 }
 
-func validateSASLConfig(c *SASLConfig) error {
+func validateSASLConfig(c *kafka.SASLConfig) error {
```

Comment:
Speaking of validation, how about defining a common validation function for `kafka.Authentication` in `internal/kafka` package, which can be reused by `kafkametricsreceiver`, `kafkareceiver` and `kafkaexporter`.

Reply:
could you help create an issue for this?